packages/remote-config/lib/index.js

```diff
@@ -232,7 +343,7 @@ export default createModuleNamespace({
   namespace,
   nativeModuleName,
   nativeEvents: false,
-  hasMultiAppSupport: false,
+  hasMultiAppSupport: true,
```

Comment:
Might have missed it but could  we add a test or two that tests multiple apps support is working correctly?